src/services/meta.ts

```diff
@@ -9,13 +9,29 @@ const META_URL = "https://meta.yearn.network";
  */
 export class MetaService extends Service {
   async tokens(addresses?: Address[]): Promise<TokenMetadata[]> {
-    const tokensMetadata = await fetch(`${META_URL}/tokens/${this.chainId}/all`).then((res) => res.json());
+    const tokensMetadata: TokenMetadata[] = await fetch(`${META_URL}/tokens/${this.chainId}/all`).then((res) =>
+      res.json()
+    );
 
     if (!addresses) {
-      return tokensMetadata;
+      return tokensMetadata.map((tokenMetadata: TokenMetadata) => ({
+        ...tokenMetadata,
+        description:
+          tokenMetadata.localization[this.ctx.locale]?.description ??
+          tokenMetadata.description ??
+          "I don't have a description for this token yet",
```

Comment:
Probably never gets here, and I know you are using the wording of the currently  used texts; would like a message like "Token description missing, contribute by adding it here [link to yearn meta]" or something like that. wdyt? (same in other places we use this missing error)

Reply:
Not sure about adding the link here since it would be a plain string and might not look very good on the ui. what about just "Token description missing"?

Reply:
Yeah you are right. Either "Token description missing" or "Token description not available" seems fine, leave it to you :)

Reply:
fixed on 70a6e8360b1facd340318999296cc6cdd7581ad0